src/worker.py

```
@@ -1329,6 +1314,354 @@ async def serve_static(path: str, env):
+        for sid, info in self.sessions.items():
+            if sid == exclude_session_id:
+                continue
+            try:
+                info["ws"].send(msg)
+            except Exception as exc:
+                print(f"[ClassroomDO._broadcast] sid={sid} pid={info.get('participant_id')} error={exc!r}")
+                dead.append(sid)
+        for sid in dead:
+            self.sessions.pop(sid, None)
+
+    def _broadcast_room_state(self):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_broadcast_room_state`

Add return type annotation: `None`

(ANN202)